File: Inboxes/Ruby/Web/RubyOnRails/src/db/migrate/20240101000001_create_messages.rb
class CreateMessages < ActiveRecord::Migration[7.1]
  def change
    create_table :messages do |t|
      t.string :sender, null: false
      t.string :subject, null: false
      t.text :body, default: ""
      t.boolean :read, default: false
      t.timestamps
    end
  end
end

File: Inboxes/Elixir/Web/Phoenix/src/priv/repo/migrations/20240101000001_create_messages.exs
defmodule Inboxes.Repo.Migrations.CreateMessages do
  use Ecto.Migration

  def change do
    create table(:messages) do
      add :sender, :string, null: false
      add :subject, :string, null: false
      add :body, :text, default: ""
      add :read, :boolean, default: false, null: false

      timestamps()
    end
  end
end